src/transform/ThemeTransform.js

```diff
@@ -2,11 +2,11 @@ import './ThemeTransform.css'
 import ElementUtilities from './ElementUtilities'
 import Polyfill from './Polyfill'
 
-// Elements marked with either of these classes indicate certain ancestry constraints that are
+// Elements marked with these classes indicate certain ancestry constraints that are
 // difficult to describe as CSS selectors.
 const CONSTRAINT = {
   IMAGE_NO_BACKGROUND: 'pagelib-theme-image-no-background',
-  IMAGE_NONTABULAR: 'pagelib-theme-image-nontabular'
+  DIV_DO_NOT_APPLY_BASELINE: 'pagelib-theme-div-do-not-apply-baseline'
```

Comment:
What do you think about using that underscore naming convention we keep talking about? I know it'd look funky in contrast with the old style but it is the way forward.

Also, consider naming the style for what its classifying, not what we want to do with it at the moment. For example, if we had a classifier that identified large houses and small houses, we could call the rules `pagelib_theme_large_house` and `pagelib_theme_small_house` and then choose to style them red and blue. Because we know we want them to be styled as red and blue, we could have also called them `pagelib_theme_red_house` and `pagelib_theme_blue_house` but it'll be brittle if we ever want a new paint job. If a small house ever becomes a large house, it will be classified as `pagelib_theme_large_house` which is what we want.

Another example that's confusing is identifying `IMAGE_NO_BACKGROUND` images with a function called `imageNeedsWhiteBackground()`. Images classified as "no background" normally _do_ need a white background but not always. When we say in the code that they need a white background, we're not consider all the inputs AND we're putting styling concerns in JavaScript. Classification is supposed to be very pointed and each class is supposed to identify one thing about an element that's difficult to describe in pure CSS. For example, there is a already a case where an image that "needs a white background" should _not_ have a white background because there are other inputs (math formulas). You can see how this was a similarly confusing case in the original Android implementation [here](https://phabricator.wikimedia.org/diffusion/APAW/browse/master/www/js/dark.js;ccc713d7abdbc9261f9ee55a4311d7645eca1783$41). `classifyElements()` is just annotating elements for what we can assert about them in JavaScript and should have no knowledge of styling.